docs/setup_config.rst

```diff
@@ -0,0 +1,58 @@
+Setup configuration
+===================
+
+Loading notification configuration from a YAML file
+***************************************************
+
+This library provides two ``ConfigurationStep`` implementations
+(from the library ``django-setup-configuration``, see the
+`documentation <https://github.com/maykinmedia/django-setup-configuration>`_
+for more information on how to run ``setup_configuration``): one to configure the 
+service and retry settings, another to configure notification endpoint subscriptions.
+
+To add these steps to your configuration steps, add ``django_setup_configuration`` 
+to ``INSTALLED_APPS`` and add the following settings:
+
+    .. code:: python
+
+        SETUP_CONFIGURATION_STEPS = [
+            ...
+            # Note the order: NotificationSubscriptionConfigurationStep expects a notification service
+            # to have been configured by NotificationConfigurationStep
+            "notifications_api_common.contrib.setup_configuration.steps.NotificationConfigurationStep"
+            "notifications_api_common.contrib.setup_configuration.steps.NotificationSubscriptionConfigurationStep"
```

Comment:
```suggestion
            "notifications_api_common.contrib.setup_configuration.steps.NotificationSubscriptionConfigurationStep" # Optional
```

Reply:
I think in the context of setting up notifications for your project through `django-setup-configuration` (e.g not manually providing data or loading in fixtures) the `NotificationSubscriptionConfigurationStep` step is optional (but requires `NotificationConfigurationStep`) and the `NotificationConfigurationStep` is required (again, assuming you only want to use `django-setup-configuration`). Not all projects will use the `NotificationSubscriptionConfigurationStep` and the current documentation might give the impression that they will have to.

If this discussion is preventing you from merging the PR, then I would say go ahead and merge it :+1: